virtac/atip_ioc_entry.py

```diff
@@ -32,15 +32,25 @@ def main():
                 value = caget("SR-CS-RING-01:MODE", format=2)
                 ring_mode = value.enums[int(value)]
             except ca_nothing:
-                ring_mode = "DIAD"
+                ring_mode = "I04"
 
     # Create PVs.
-    server = atip_server.ATIPServer(
+    if sys.argv[2:]:
```

Comment:
There isn't any clue as to what sys.argv[2:] is or what it's used for. It's probably an indication that we need to use a proper argument parsing library like argparse.

At least for this merge request, I would suggest:
- Using a specific variable or comment to indicate the purpose of sys.argv[2:]
- Providing the default value in this script, rather than relying on the default set in ATIPServer

Reply:
See other comments, I've decided you should just move to using argparse.